src/test/java/org/gridsuite/network/map/NetworkMapControllerTest.java

```diff
@@ -719,17 +720,25 @@ private String resourceToString(String resource) throws IOException {
         return new String(ByteStreams.toByteArray(getClass().getResourceAsStream(resource)), StandardCharsets.UTF_8);
     }
 
-    private String buildUrlForList(String equipments, String variantId, List<String> immutableListSubstationIds) {
+    private String buildUrlForList(String equipment, String variantId, List<String> immutableListSubstationIds, boolean onlyIds) {
```

Comment:
`equipmentTypeInfo` in place of `equipment` ?

Reply:
I named it equipmentType to be consistent with the others